open_city_profile/tests/test_translation_middleware.py

```diff
@@ -0,0 +1,50 @@
+import graphene
+import pytest
+from django.conf import settings
+from django.utils import translation
+from django.utils.translation import gettext
+from graphene.test import Client
+from graphene_federation import build_schema
+from graphql import specified_directives
+
+from open_city_profile.graphene import (
+    TranslationDirective,
+    TranslationLanguageType,
+    TranslationMiddleware,
+)
+
+
+@pytest.mark.parametrize("language", [lang[0] for lang in settings.LANGUAGES])
+def test_translation_middleware_changes_language_for_field(user_gql_client, language):
```

Comment:
This test doesn't use the `user_gql_client`.

Reply:
Nice catch.